flood_adapt/object_model/hazard/hazard.py

```diff
@@ -259,17 +266,14 @@ def postprocess_models(self):
         # Postprocess all hazard model input
         self.postprocess_sfincs()
         # add other models here
-        # WITHOUT A SFINCS FOLDER WE CANNOT READ sfincs_map.nc ANYMORE
         # remove simulation folders
-        # if not self.site.attrs.sfincs.save_simulation:
-        #     for simulation_path in self.simulation_paths:
-        #         if os.path.exists(simulation_path.parent):
-        #             try:
-        #                 shutil.rmtree(
-        #                     simulation_path.parent
-        #                 )  # TODO cannot remove simulation because hydromt log file is still being used
-        #             except WindowsError:
-        #                 pass
+        if not self.site.attrs.sfincs.save_simulation:
+            sim_path = self.results_dir.joinpath("simulations")
+            if os.path.exists(sim_path):
+                try:
+                    shutil.rmtree(sim_path)
+                except WindowsError:
+                    pass
```

Comment:
This works, but it would be better to avoid writing and deleting files where possible.
If we can add a check before writing large files to disk that are not going to be used by other parts of the codebase, we can save more time. 
Its not possible everywhere though since other models might use the created files so definitely keep this.